src/to_geojson.rs

```diff
@@ -47,6 +49,15 @@ fn make_pt(pos: &[f64], tr: &Option<TransformParams>) -> Vec<f64> {
     }
 }
 
+fn make_feature_id(id: Option<JsonValue>) -> Option<FeatureId> {
+    match id {
+        Some(JsonValue::Number(i)) => Some(FeatureId::Number(i)),
+        Some(JsonValue::String(i)) => Some(FeatureId::String(i)),
+        Some(i) => Some(FeatureId::String(String::from(i.as_str().unwrap()))),
```

Comment:
I'm not really sure about this, i don't think we should unwrap this blindly.
It's not supposed to be the case but if the `id` is whether an Array, an Object, a Boolean or a null value, this will panic without any information.
As it's not supposed to happen, it might be better to return an `Error` with a meaningful description if the id is neither a Number nor a String. Maybe something like:

``` rust
fn make_feature_id(id: Option<JsonValue>) -> Result<Option<FeatureId>, Error> {
    match id {
        Some(JsonValue::Number(i)) => Ok(Some(FeatureId::Number(i))),
        Some(JsonValue::String(i)) => Ok(Some(FeatureId::String(i))),
        None => Ok(None),
        _ => Err(Error::InvalidIdType)
    }
}
```

Or maybe it makes more sense to still return directly an `Option` instead of wrapping it in a `Result`. If so I think it should probably returns `None` for not-handled cases (and warn the user that the `id` field wasn't used).

Reply:
That makes sense to me. It seems like the `id` being something other than a Number or String would be relatively rare, and since it's optional in GeoJSON defaulting to `None` seems relatively safe. What's the best way of logging a warning for that, and should it log once for each feature where the `id` was dropped?

Reply:
The [log](https://docs.rs/log/) crate (and its `warn!` macro) used with the [`env_logger`](https://docs.rs/env_logger/) crate seems to be a popular choice (and I honestly don't have much experience with the other options!).
Personally I will be fine with having it logged for each where the `id` was dropped (and it might be simpler to implement) but it's up to you.

Reply:
Great, thanks! The [`env_logger` repo](https://github.com/sebasmagri/env_logger) says it's for binaries, so I just used the `log` crate here. I imported it into the file directly since it's only used here, but I can move that to `lib.rs` if it works better

Reply:
Good catch for the env_logger ;)